src/bdscript/serverDescription.md

```diff
@@ -0,0 +1,14 @@
+# $serverDescription
+Returns a description of the server. Returns an empty result if the server has no description.
+> 🧙‍♂️ How to set the server description? The server description can be set in the server settings if Community is enabled on your server. [More (Click)](https://support.discord.com/hc/en-us/articles/360047132851-Enabling-Your-Community-Server)
```

Comment:
```suggestion
> 🧙‍♂️ How to set the server description? It can be set in the server settings if Community is enabled on your server. [More (Click)](https://support.discord.com/hc/en-us/articles/360047132851-Enabling-Your-Community-Server)
```